frontend/src/components/pages/home/Home.tsx

```diff
@@ -56,10 +56,12 @@ const HomeUI: React.FC = () => {
             .filter(({ results }) => results.length > 0)
             .map(({ titleTranslationId, iconUrl, results }) => (
               <HomeSection
+                className="mx-4 desktop:mx-10percent"
                 title={intl.formatMessage({ id: titleTranslationId })}
                 iconUrl={iconUrl}
                 key={titleTranslationId}
                 results={results}
+                asColumn
```

Comment:
As default value of `asColumn` prop in `HomeSection` is `true`, maybe would it preferable to not specify it here.

_(This will also avoid wrongly thinking: "if I remove this prop here, then it's not asColumn anymore")_

Reply:
You're totally right! :+1: 
I should use this component later in another place with `asColumn` set to `false`.
I'll change the default value to `false` to avoid confusion.

Reply:
Done!